spec/requests/vendor_api/post_confirm_enrolment_spec.rb

```diff
@@ -31,6 +31,6 @@
 
     expect(response).to have_http_status(:not_found)
     expect(parsed_response).to be_valid_against_openapi_schema('NotFoundResponse')
-    expect(error_response['message']).to eql('Could not find an application with ID non-existent-id')
+    expect(error_response['message']).to eql('Unable to find Application(s)')
```

Comment:
Update to match documentation: https://www.apply-for-teacher-training.service.gov.uk/api-docs/reference#notfoundresponse-object